Arrowgene.Ddon.Shared/Network/PacketId.cs

```diff
@@ -1054,11 +1054,11 @@ public override int GetHashCode()
         public static readonly PacketId S2C_SKILL_GET_ABILITY_COST_RES = new PacketId(19, 39, 2, "S2C_SKILL_GET_ABILITY_COST_RES", ServerType.Game, PacketSource.Server); // アビリティセット用コストの取得に
         public static readonly PacketId C2S_SKILL_GET_PAWN_ABILITY_COST_REQ = new PacketId(19, 40, 1, "C2S_SKILL_GET_PAWN_ABILITY_COST_REQ", ServerType.Game, PacketSource.Client);
         public static readonly PacketId S2C_SKILL_GET_PAWN_ABILITY_COST_RES = new PacketId(19, 40, 2, "S2C_SKILL_GET_PAWN_ABILITY_COST_RES", ServerType.Game, PacketSource.Server); // ポーンアビリティセット用コストの取得に
-        public static readonly PacketId S2C_SKILL_19_41_16_NTC = new PacketId(19, 41, 16, "S2C_SKILL_19_41_16_NTC", ServerType.Game, PacketSource.Server);
-        public static readonly PacketId S2C_SKILL_19_42_16_NTC = new PacketId(19, 42, 16, "S2C_SKILL_19_42_16_NTC", ServerType.Game, PacketSource.Server);
+        public static readonly PacketId S2C_SKILL_ACQUIREMENT_LEARN_NTC = new PacketId(19, 41, 16, "S2C_SKILL_ACQUIREMENT_LEARN_NTC", ServerType.Game, PacketSource.Server); //Probably, based on order in debug symbols.
```

Comment:
Add ` S2C_SKILL_19_41_16_NTC` and `S2C_SKILL_19_42_16_NTC` as the last argument to `new PacketId`. Do the same for all the renamed packets.